js/modules/k6/http/http_request.go

```diff
@@ -235,7 +248,10 @@ func (h *HTTP) request(ctx context.Context, rt *goja.Runtime, state *common.Stat
 		}
 	}
 
-	resp := &HTTPResponse{ctx: ctx, URL: url.URLString}
+	HTTPReq.Headers = req.Header
+	HTTPReq.Cookies = reqCookies
```

Comment:
The cookies sent with the request could also be retrieved from the active cookiejar so we need similar logic to what is in https://github.com/loadimpact/k6/blob/master/js/modules/k6/http/http.go#L130 (called a right above this) to determine the actual cookies sent. I guess the easiest would be to have `setRequestCookies()` return the determined set of cookies to send (`jarCookies`).

Reply:
Not sure I understand completely. I think you're saying we want to add `getRequestCookies()` to get the cookies that would have actually have been put into the `http.Request` struct?